content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -1,139 +1,151 @@
-* Enter your TaxCloud credentials.
-* Click SAVE to store your credentials.
-
-.. image:: taxcloud/taxcloud02.png
-  :align: center
-
-* Click the Refresh Icon next to *Default Category* to import the TIC product categories
-  from TaxCloud (Taxability Information Codes). Some categories may imply specific rates or attract product/service type exemptions.
-* Select your default *TIC Code*. This will apply to any new
-  product created.
-* Set a specific TaxCloud TIC Category on the *General Information* tab of the Product,
-  or on the Product Category.
-* Make sure your company address is complete (including the state
-  and the zip code). Go to :menuselection:`Settings --> Users & Companies --> Companies`
-  to open and edit your Company record.
-
-Automatically post taxes to the correct Tax Payable account
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-* For each one of your companies that uses TaxCloud, it might be necessary to create a 
-  **User-Defined Default** so that the new taxes generated by the TaxCloud integration are created 
-  with the correct Tax Payable account:
-  
-.. warning::
-   A User-Defined Default impacts all records at creation.  It means that **every** new tax will be 
-   set up to record income in the specified Tax Payable account, unless the tax is manually edited
-   to specify a different income account (or there exists another User-Defined Default that takes 
-   precedence).
+====================
+TaxCloud integration
+====================
 
-* In :menuselection:`Accounting --> Configuration --> Chart of Accounts`, select the Tax Payable
-  account for the company. Take note of the account's ``id`` in the URL string. 
+TaxCloud calculates the sales tax rate in real time for every state, city, and special jurisdiction
+in the United States. It keeps track of which products are exempt from sales tax and in which states
+each exemption applies.
 
-.. image:: taxcloud/user-default-find-account-id.png
-   :alt: The account's ID can be found in the URL string as 'id=...'. 
+TaxCloud registration
+=====================
+
+Register an account on `TaxCloud.com <https://taxcloud.com/register>`_ and complete the setup.
+Once you go live, get the :guilabel:`TaxCloud API Keys` by clicking on :guilabel:`Stores`, then
+:guilabel:`Get Details`.
+
+.. image:: taxcloud/taxcloud-api-keys.png
    :align: center
+   :alt: Example of a store's TaxCloud API Keys
+
+Enable TaxCloud
+===============
+
+#. Go to :menuselection:`Accounting Dashboard --> Configuration --> Settings` and in the
+   :guilabel:`Taxes` section enable :guilabel:`TaxCloud`.
+#. Add the store's :guilabel:`Login ID` under :guilabel:`API ID` and the store's :guilabel:`Key`
+   under :guilabel:`API KEY`. Click on :guilabel:`Save`.
+#. Click the :guilabel:`Refresh` button (:guilabel:`🗘`) next to :guilabel:`Default Category` to
+   import the :dfn:`TIC Taxability Information Codes` product categories from TaxCloud. Some
```

Comment:
My bad, I didn't write my suggestion properly. The `dfn` markup applies styling on everything between the backticks.
```suggestion
   import the TIC :dfn:`Taxability Information Codes` product categories from TaxCloud. Some
```